cmd/src-fingerprint/main.go

```diff
@@ -137,6 +137,11 @@ func main() {
 				Required: true,
 				Usage:    "vcs provider. options: 'gitlab'/'github'/'bitbucket'/'repository'",
 			},
+			&cli.BoolFlag{
+				Name:  "all",
+				Value: false,
+				Usage: "Collect hashes from every accessible repository, not just private repositories if the provider is 'github'",
```

Comment:
I would change a bit the wording, and avoid using the term hashes. Why not something like : "Collect fileshas from every accessible repository including  public ones. This option has an effect only on github provider."